_docs_integrate/25-requests-over-templates.md

```diff
@@ -0,0 +1,158 @@
+
+```jsonc
+{
+    "items": [
+        {
+            "accept": false
+        }
+    ]
+}
+```
+
+You receive a Request in status `Decided`. This is where the Request module steps in and handles the Request based on you decision. Because you rejected the Request, the Request module will only move the Request to status completed. This behavior can be observed by querying the Request again after a short waiting time (`GET /api/v2/Requests/Incoming/{id}`).
+
+### Accept
+
+If you want to accept the Request you can do so by calling the `POST /api/v2/Requests/Incoming/{id}/Accept` route. You can use the `id` you saved in the previous step. In the payload you have to accept all RequestItems where the `mustBeAccepted` property is set to `true`. In case of the example Request the payload is the following:
+
+```jsonc
+{
+    "items": [
+        {
+            "accept": true
+        }
+    ]
+}
+```
+
+You receive a Request in status `Decided`. This is where the Request module steps in and handles the Request based on you decision. Because you accepted the Request, the Request module will send your Response to the Templator by creating a Relationship. This behavior can be observed by querying the Request again after a short waiting time (`GET /api/v2/Requests/Incoming/{id}`). When the Request is in status `Completed` you can query the created Relationship (`GET /api/v2/Relationships`, param `template.id=<id-of-the-template>`).
+
+If you sync the Templator Connector you will see a new Relationship in the Response. The Relationship looks as follows:
+
+```jsonc
+{
+    "id": "REL...",
+    "template": {
+        // ...
+    },
+    "status": "Active",
+    "peer": "id1...",
+    // ...
+    "changes": [
+        {
+            "id": "RCH...",
+            "request": {
+                "createdBy": "id1...",
+                "createdByDevice": "DVC...",
+                "createdAt": "2022-11-04T13:31:01.360Z",
+                "content": {
+                    "@type": "RelationshipCreationChangeRequestContent",
+                    "response": {
+                        "items": [
+                            {
+                                "@type": "AcceptResponseItem",
+                                "result": "Accepted"
+                            }
+                        ],
+                        "requestId": "REQ...",
+                        "result": "Accepted"
+                    }
+                }
+            },
+            "status": "Accepted",
+            "type": "Creation"
+            // ...
+        }
+    ]
+}
+```
+
+Especially pay attention for the `changes.0.request.content` property that was generated by the Request module from your Response.
+
+Now you can accept the Relationship by calling the `PUT /api/v2/Relationships/{relationshipId}/Changes/{changeId}/Accept` route. You can now sync the Requestor Connector and see that the Relationship is now in status `Active` on both Connectors.
```

Comment:
> You can now sync the Requestor Connector and see that the Relationship is now in status `Active` on both Connectors.

Das sollte in ein eigenes Kapitel. Weil aktuell steht es unter "Answer the Request > Accept", was ja falsch ist.